test/mitmproxy/data/addonscripts/configure.py

```diff
@@ -0,0 +1,27 @@
+import typing
+
+from mitmproxy import ctx
+from mitmproxy import exceptions
+
+
+class TestHeader:
```

Comment:
Give this thing a better name. :)

Reply:
I changed the name to "OptionAddon" if that doesn't fit can you give me a little hint, I am not too good with names.